tests/auto/rt_auto.py

```diff
@@ -0,0 +1,474 @@
+# HEADER
```

Comment:
There needs to be some documentation as to what each class, function does.

Reply:
Working on this, but will add all before merge